src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * List Widget
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Widget\Widget_Abstract;
+
+/**
+ * Class for the List Widget.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Widget_List extends Widget_Abstract {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $slug = 'tribe_events_list_widget';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $view_slug = 'widget-list';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $default_arguments = [
+		// View options.
+		'view'                 => null,
+		'should_manage_url'    => false,
+
+		// Event widget options.
+		'id'                   => null,
+		'alias-slugs'          => null,
+		'title'                => '',
+		'limit'                => 5,
+		'no_upcoming_events'   => '',
+		'featured_events_only' => false,
+		'tribe_is_list_widget' => true,
+		'jsonld_enable'        => true,
+
+		// WP_Widget properties.
+		'id_base'              => 'tribe-events-list-widget',
+		'name'                 => null,
+		'widget_options'       => [
+			'classname'   => 'tribe-events-list-widget',
+			'description' => null,
+		],
+		'control_options'      => [
+			'id_base' => 'tribe-events-list-widget',
+		],
+	];
+
+	/**
+	 * @todo update in TEC-3612 & TEC-3613
+	 *
+	 * {@inheritDoc}
+	 */
+	protected $validate_arguments_map = [
```

Comment:
:no_entry_sign: Missing @var tag in member variable comment

<sub>hash: 76d863185f28ba04274b381433102dcc</sub>